controller/update_test.go

```diff
@@ -0,0 +1,339 @@
+package controller_test
+
+import (
+	"context"
+	"fmt"
+	goatest "github.com/fabric8-services/fabric8-tenant/app/test"
+	"github.com/fabric8-services/fabric8-tenant/configuration"
+	"github.com/fabric8-services/fabric8-tenant/controller"
+	"github.com/fabric8-services/fabric8-tenant/environment"
+	"github.com/fabric8-services/fabric8-tenant/tenant"
+	"github.com/fabric8-services/fabric8-tenant/test"
+	"github.com/fabric8-services/fabric8-tenant/test/doubles"
+	"github.com/fabric8-services/fabric8-tenant/test/gormsupport"
+	tf "github.com/fabric8-services/fabric8-tenant/test/testfixture"
+	"github.com/fabric8-services/fabric8-tenant/test/update"
+	"github.com/fabric8-services/fabric8-tenant/update"
+	"github.com/fabric8-services/fabric8-wit/ptr"
```

Comment:
you can use ptr from common here.

Reply:
slipped out of my notice, thanks - fixed 831bfcf